doofinder-for-woocommerce/includes/settings/class-attributes.php

```diff
@@ -184,21 +192,31 @@ private function get_attribute_meta( $source, $product ) {
 	 * @return mixed The attribute value.
 	 * @since 1.2.2
 	 */
-	private function get_attribute_wc( $source, $product ) {
+	private function get_attribute_wc($source, $product)
```

Comment:
I've changed this function that was obtaining variation attributes from a string and exploding...
Now they're obtained with the function WC_Variation->get_variation_attributes()